packages/api/hardware-checkout/src/services/hardware.mjs

```diff
@@ -0,0 +1,96 @@
+import database from '../../../core/src/utils/database.js';
+
+const hardware = (app) => {
+    
+    //GET operatiion to return a piece of hardware by unique ID 
+    //This function executes a stored procedure the returns a JSON object containing details about the hardware item 
+    //const scheme is the JSON object that is returned containing the details of the hardware item
+    app.get('/hardware/:id', async (req, res) => {
+        const hardware = await database.executeStoredProcedure('Select_Single_Hardware_Items', req.params.id);
```

Comment:
The way we binded the parameters to the stored procedure in the executeStoredProcedure function is considered bad practice and is actually a security vulnerability. This is something we can fix later, for now we should avoid using that function. Instead try this:
```
const session = await database.connect();

await session.sql('SET @id = ?;')
            .bind(req.params.id)
            .execute();

const statement = "CALL Select_Single_Hardware_Items(@id)";
const result = await session.sql(statement).execute();
const hardware = await result.fetchOne();
```